tests/pr_test.go

```diff
@@ -2,30 +2,59 @@
 package test
 
 import (
+	"log"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/common"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
 const defaultExampleTerraformDir = "examples/default"
+const completeExampleTerraformDir = "examples/complete"
+
+// Use existing group for tests
+const resourceGroup = "geretain-test-event-streams"
+
+// Not all regions provide cross region support so value must be hardcoded https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-service-availability.
+const region = "us-south"
```

Comment:
since we are not using region here, remove it from the code (and the comment above it)